alert/templates/application/home.html

```diff
@@ -1,44 +1,15 @@
-{% extends "layouts/base.html" %}
+{% extends "layouts/application.html" %}
 
+{% block content %}
 
-{% block app_content %}
+<div class="row">
+  <div class="col">
+    {% include "application/components/info_panel.html" %}
+  </div>
 
-    <div class="container-fluid" >
+  <div class="col-9">
+    {% include "application/components/main_panel.html" %}
+  </div>
+</div>
 
-        <div class="col-xs-3" style="background-color:aqua; min-height:80vh">
-            context about grammars (couldn't get accordion to work here 😔)
```

Comment:
No worries, flask bootstrap can be fiddly. Using the standard CDN bootstrap meant I could copy/paste the examples from the docs. I'll let you customise the content.